pep-0715.rst

```diff
@@ -20,7 +20,10 @@ This PEP recommends deprecating and then disabling new uploads of the
 deprecating and then disabling new uploads of distribution filenames that have
 the ``.egg`` suffix.
 
-It does not recommend removing or otherwise affecting any previously
+After this PEP, PyPI will only accept new uploads of the ``sdist``
+and ``bdist_wheel`` types.
```

Comment:
Maybe also worth adding the accepted suffices, per @pfmoore 's [comment](https://discuss.python.org/t/pep-715-disabling-bdist-egg-distribution-uploads-on-pypi/27610/2?u=cam-gerlach) on the PEP discussion thread?

Reply:
Good point -- I totally misread the originally comment as being about filetypes, not extensions. Making a PR now.

Reply:
#3169

Reply:
Thanks!